.drone.yml

```diff
@@ -4,14 +4,14 @@ workspace:
 
 services:
   web:
-    image: fpfis/httpd-php-ci:${PHP_VERSION=7.1}
+    image: fpfis/httpd-php-ci:${PHP_VERSION=7.2}
```

Comment:
The variable is always defined, let's remove the default value.

Reply:
The default value is used if you want to test your pipeline with drone-cli in your local machine.
Drone-cli doesn't use the matrix.

Reply:
@brummbar this is common practice in all other components.
If you still think and disagree with input from @voidtek I could remove but let's leave this for at least for consistance.

Reply:
We talked with @ademarco and decided to drop it.

Reply:
Done